xrpl4j-model/src/main/java/org/xrpl/xrpl4j/model/client/server/ServerInfoLedger.java

```diff
@@ -66,16 +77,36 @@ static ImmutableServerInfoLedger.Builder builder() {
    *
    * @return An {@link UnsignedInteger} representing the amount of XRP to reserve.
    */
+  @JsonIgnore
+  @Value.Auxiliary
+  @Deprecated
+  @Value.Default
+  default UnsignedInteger reserveBaseXrp() {
+    return UnsignedInteger.valueOf(reserveBaseDrops().toXrp().setScale(0, RoundingMode.CEILING).toBigInteger());
+  }
+
   @JsonProperty("reserve_base_xrp")
-  UnsignedInteger reserveBaseXrp();
+  @JsonSerialize(using = DecimalXrpCurrencyAmountSerializer.class)
+  @JsonDeserialize(using = DecimalXrpCurrencyAmountDeserializer.class)
+  XrpCurrencyAmount reserveBaseDrops();
 
   /**
    * Amount of XRP (not drops) added to the account reserve for each object an account owns in the ledger.
    *
    * @return An {@link UnsignedInteger} representing the amount of XRP added.
    */
+  @JsonIgnore
+  @Value.Auxiliary
+  @Deprecated
```

Comment:
Let's add a `@deprecated` to the javadoc explaining why this is deprecated. Let's also update the javadoc to say that this value will be rounded.